s4e-backend/src/test/java/pl/cyfronet/s4e/admin/product/AdminProductControllerTest.java

```diff
@@ -390,6 +392,13 @@ public void shouldVerifyInService() throws Exception {
 
         @BeforeEach
         public void beforeEach() {
+            StreamSupport
+                    .stream(productCategoryRepository.findAll().spliterator(), false)
+                    .filter(productCategory -> !productCategory.getLabel().contains(ProductCategoryRepository.DEFAULT_CATEGORY_LABEL))
+                    .peek(productCategory -> productCategoryRepository.delete(productCategory))
+                    .collect(Collectors.toList());
+
+            val category = productCategoryRepository.save(ProductCategoryHelper.productCategoryBuilder().build());
             product = productRepository.save(Product.builder()
```

Comment:
This test class should be extended to check, if the new relation is handled well, and that you can modify a category of a product.